.github/ISSUE_TEMPLATE/release-team-lead-progress.md

```diff
@@ -0,0 +1,241 @@
+- [ ] Discussed Release Lead succession with the EA and sig-release leads
+- [ ] Remind team lead to find successors for the upcoming cycle and discuss candidates with the EA
+- [ ] The task is now to ensure the release branch is ready to go. This means there are zero pending PRs, no failing 1.XX-blocking tests, no open issues in the milestone. This will continue until release day.
+- [ ] Final documentation PRs are reviewed and ready to be merged. Likely, this is not true and some are outstanding, so you need to help convince SIG doc writers to get these in with urgency.
+- [ ] Planned something for Release Day. Make the day as fun as you can for the team. Plan ahead for this and do something nice
+- [ ] Prepared [Release Team gifts](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/release-team-lead#release-theme-gifts)
+- [ ] The Communications Lead contacts CNCF to gauge media interest, schedule the CNCF Kubernetes release workshop, and publish the release blog post. Stay in the loop for that. If there is media interest in the release, an interview between the journalist will be organized by the CNCF.
+- [ ] The Release Team Retro part 2 and part 3 is scheduled shortly after the "Release Halfway Point" and a host / facilitator is available
+- [ ] Remind the release team to add items to the retro meeting agenda
+
+### 7. Release Day (~Week 15)
+
+- Note that release day can and should be postponed if any of the conditions outlined in week 11 are not satisfied.
+- Every issue in the milestone is considered release blocking
+- If you have to push the release date back, try to avoid Friday since it makes release publicity extremely difficult. Also, people seem to have patience with delay as long as the reasons are clear and openly communicated. This is your duty. You must over-communicate and ensure the team is also talking to their stakeholders (CNCF, community, press, etc.)
+
+- The following final actions **must occur in order**, with successful completion of each being the entry criteria to the next.
+  - Release day morning:
+    - [ ] Go / No-Go: should generally be clear a day or three ahead of release, but the day's burndown provides a final opportunity for the team to affirm things are ready.
+    - [ ] Release Notes Lead PRs final draft release notes to sig-release, with Release Team Lead approving merge.
+    - [ ] Branch Manager does mock release build.
+    - [ ] Branch Manager does mock publication. Validates with Release Team lead and broader team the mock announcement email content.
+    - [ ] Branch Manager does nomock release build.
+  - Starting when ready:
+    - [ ] Communications Lead begins staging blog post.
+    - [ ] Branch Manager Lead does nomock publication.
+    - [ ] Branch Manager coordinates the building and publishing of rpm/deb packages with Google Build Admins.
+    - [ ] Build Admins affirms build is complete.
+    - [ ] Docs Lead publishes release docs to website.
```

Comment:
I'd mention that this is a _manual merge_ due to the freeze.

Reply:
OK to leave as-is and fix in a later PR, though.

Reply:
We could link to the docs handbook for more details maybe? https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/docs/Release-Timeline.md#release-week-week-12

Reply:
added the link ✅